src/pages/search/components/NoResults.jsx

```diff
@@ -12,7 +12,7 @@ function NoResults(props) {
     <div className='flexWrapper'>
       <div className='verticalAlign'>
         <h3>{ helperHeader }</h3>
-        <p>Trying searching for something, or</p>
+        <p>Try searching for something else, or</p>
```

Comment:
Please remove the `else` or add some logic so it reads well with the options for `helperHeader`

Reply:
This one is tricky. I did think about this a little bit.

In the case that there is no search input, I thought that "Try searching for something" could come across as a sarcastic remark (don't know what that says about me 😅). 

I'm not sure what would be ideal here. How about "Please try searching for something using the search bar above, or" for the empty input cases?

Reply:
How about `You can try searching for something, or`?

Reply:
Sorry, I was being too perfectionistic 😅